addons/website/views/snippets/snippets.xml

```diff
@@ -327,6 +327,10 @@
                 </t>
                 <t t-snippet="website.s_opening_hours" string="Opening Hours" group="contact_and_forms"/>
                 <t t-snippet="website.s_contact_info" string="Contact Info" group="contact_and_forms"/>
+                <t t-snippet="website.s_website_form" string="Form" t-forbid-sanitize="form" t-thumbnail="/website/static/src/img/snippets_thumbs/s_website_form.svg" group="contact_and_forms"/>
```

Comment:
Check if necessary, it seems to be a leftover diff of the inner form snippet 😇

Reply:
Good catch thanks ! I updated the branch 👍 
The diff: https://github.com/odoo/odoo/compare/2a48cbfb8299cfda3f690499fad208b43d1c2a87..9b198aab30cf52f26684762f80f61f30a0094ddd